cli/script/command-executor.ts

```diff
@@ -39,16 +39,18 @@ interface NameToCountMap {
     [name: string]: number;
 }
 
+/** Deprecated */
 interface ILegacyLoginConnectionInfo {
     accessKeyName: string;
-    providerName: string;
-    providerUniqueId: string;
-    serverUrl: string;
+    // The 'providerName' property has been obsoleted
+    // The 'providerUniqueId' property has been obsoleted
+    // The 'serverUrl' property has been obsoleted
 }
 
 interface ILoginConnectionInfo {
     accessKey: string;
```

Comment:
What do you think about making this be called "accessKeyName" instead, then we can merge the two interfaces, and not have to care whether the saved one old or the new format?